src/DssSpell.sol

```diff
@@ -44,109 +40,146 @@ contract DssSpellAction is DssAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
     // uint256 internal constant X_PCT_RATE = ;
+    uint256 internal constant SIX_PT_FOUR_FIVE_PCT_RATE  = 1000000001982027061559507021;
+    uint256 internal constant SIX_PT_FOUR_NINE_PCT_RATE  = 1000000001993940198563273844;
+    uint256 internal constant SIX_PT_SEVEN_PCT_RATE      = 1000000002056410844314321266;
+    uint256 internal constant SIX_PT_SEVEN_FOUR_PCT_RATE = 1000000002068296073857195778;
+    uint256 internal constant SIX_PT_NINE_ONE_PCT_RATE   = 1000000002118758660201099744;
+    uint256 internal constant SEVEN_PT_ONE_SIX_PCT_RATE  = 1000000002192822766493423465;
+    uint256 internal constant SEVEN_PT_TWO_PCT_RATE      = 1000000002204656986467871801;
+    uint256 internal constant SEVEN_PT_TWO_FOUR_PCT_RATE = 1000000002216486791512316847;
 
     // ---------- Math ----------
-    uint256 internal constant MILLION  = 10 **  6;
-
-    address internal immutable MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
-
-    // Note: Function from https://github.com/makerdao/spells-goerli/blob/7d783931a6799fe8278e416b5ac60d4bb9c20047/archive/2022-11-14-DssSpell/Goerli-DssSpell.sol#L59
-    function _updateDoc(bytes32 ilk, string memory doc) internal {
-        ( , address pip, uint48 tau, ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
-        require(pip != address(0), "DssSpell/unexisting-rwa-ilk");
-
-        // Init the RwaLiquidationOracle to reset the doc
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
-            ilk, // ilk to update
-            0,   // price ignored if init() has already been called
-            doc, // new legal document
-            tau  // old tau value
-        );
-    }
+    uint256 internal constant THOUSAND = 10 ** 3;
+    uint256 internal constant RAY      = 10 ** 27;
+    uint256 internal constant RAD      = 10 ** 45;
+
+    // ---------- Reduce PSM-PAX-A Debt Ceiling & Disable DC-IAM ----------
+    VatAbstract internal immutable vat = VatAbstract(DssExecLib.vat());
+
+    // ---------- RETH-A Offboarding Parameters Finalization ----------
+    address internal immutable MCD_SPOT = DssExecLib.spotter();
+
+    // ---------- SBE parameter changes ----------
+    address internal immutable MCD_VOW  = DssExecLib.vow();
+    address internal immutable MCD_FLAP = DssExecLib.flap();
```

Comment:
I'm not sure the comments in the declarations is a good thing to have.
While it's not the case in the current spell, usually specific contracts might be used in more than 1 spell action, making it more cumbersome to declare.

Reply:
The comments here are just a way to structure declarations. We can also group them by type instead of the content sections, but I think the end goal of both approaches is similar: increased readability. In the archive I can see both approaches mixed. Unless we want to bureaucratise this, I would keep it as is 🙂